testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/ProtobufPluginSmokeTest.groovy

```diff
@@ -26,6 +26,9 @@ import static org.gradle.testkit.runner.TaskOutcome.UP_TO_DATE
 
 class ProtobufPluginSmokeTest extends AbstractPluginValidatingSmokeTest {
 
+    // https://central.sonatype.com/artifact/com.google.protobuf/protobuf-java/versions
+    private static protobufToolsVersion = "4.31.1"
```

Comment:
💅 `4.32.1` could we automate this as well using https://repo1.maven.org/maven2/com/google/protobuf/protobuf-java/maven-metadata.xml?

Reply:
I thought about it, but I don't think it's worth automating. Second, we are testing the plugin, not the tools. If the plugin breaks, this can be updated manually.